spec/lib/code_ownership/cli_spec.rb

```diff
@@ -53,6 +53,43 @@
           MSG
           subject
         end
+
+        context 'when file is annotated' do
```

Comment:
The `before` blocks are a bit verbose, as are the redundant `it` blocks. I can clean them up with RSpec shared examples, etc, if this PR seems worth merging.